packages/utils/README.md

```diff
@@ -0,0 +1,37 @@
+<h1 align="center">Utils for ESLint Plugins</h1>
+
+<p align="center">Utilities for working with TypeScript + ESLint together.</p>
+
+<p align="center">
+    <img src="https://github.com/typescript-eslint/typescript-eslint/workflows/CI/badge.svg" alt="CI" />
+    <a href="https://www.npmjs.com/package/@typescript-eslint/eslint-plugin"><img src="https://img.shields.io/npm/v/@typescript-eslint/eslint-plugin.svg?style=flat-square" alt="NPM Version" /></a>
+    <a href="https://www.npmjs.com/package/@typescript-eslint/eslint-plugin"><img src="https://img.shields.io/npm/dm/@typescript-eslint/eslint-plugin.svg?style=flat-square" alt="NPM Downloads" /></a>
+</p>
+
+## Note
+
+This package has inherited its version number from the `@typescript-eslint` project.
+Meaning that even though this package is `2.x.y`, you shouldn't expect 100% stability between minor version bumps.
+i.e. treat it as a `0.x.y` package.
+
+Feel free to use it now, and let us know what utilities you need or send us PRs with utilities you build on top of it.
+
+Once it is stable, it will be renamed to `@typescript-eslint/util` for a `6.0.0` release.
```

Comment:
![](https://c.tenor.com/U9Q6lSEgWtkAAAAM/umm-um.gif)

Reply:
_This_ time it'll happen!

Reply:
Haha!

Btw that line does have a typo in it (missing the `s` on utils)